control_msgs/srv/SetConfig.srv

```diff
@@ -0,0 +1,4 @@
+string config_name
```

Comment:
How do you configure just based on string?. I think It is missing some documentation